FRONTEND_DOCUMENTATION.md

```diff
@@ -0,0 +1,12 @@
+# FRONTEND DOCUMENTATION
+
+## FRAMEWORK
+* Angular node JS
+
+## WHAT THE SITE WILL LOOK LIKE
+The following images are what our site login page and main page are planned to look like.
+
+![Login_page](/Users/student/Desktop/NYUAD Alumni Website Project/login.png)
```

Comment:
Unfortunately, this image path `/Users/student/Desktop/NYUAD Alumni Website Project/login.png` will only work on your computer. I suggest we store the images online (GDrive or Dropbox) and use the public links here instead :)